spec/services/prison_service_spec.rb

```diff
@@ -6,6 +6,10 @@
     expect(name).to eq('HMP Leeds')
   end
 
+  it 'can find all womens prisons' do
+    expect(described_class::WOMENS_PRISON_CODES.size).to eq(12)
+  end
```

Comment:
Is it worth testing the behaviour of `PrisonService.womens_estate?` in this spec, too?
i.e. the change in behaviour when the feature flag is on/off

I don't have a strong opinion either way 🤷🏻 we may be fine with the 'implicit' test in the routing spec.